__tests__/app.test.js

```diff
@@ -225,6 +225,88 @@ describe('ENDPOINT TESTS', () => {
                 })
         });
     });
+    describe('PATCH /api/articles/:article_id ', () => {
+        test('PATCH 200 - responds with the updated article', () => {
+            return request(app)
+                .patch('/api/articles/2')
+                .send({ inc_votes: 10 })
+                .expect(200)
+                .then((res) => {
+                    expect(res.body.article).toEqual(
+                        expect.objectContaining({
+                            article_id: expect.any(Number),
+                            title: expect.any(String),
+                            topic: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                        })
+                    );
+                    const dateCreated = new Date(res.body.article.created_at)
+                    expect(dateCreated).toBeInstanceOf(Date)
+                    expect(res.body.article.votes).toBeGreaterThanOrEqual(10)
+                });
+        });
```

Comment:
It would be more thorough to add a test here for inc_votes being a negative number